.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 18.18.0
```

Comment:
shouldn't this be node v20 (the current lts)?

Reply:
Yeah I am using 18.18.0 on my work machine but I can add the LTS and update this file.

Reply:
Updated to 20.10.0